### ISSUE TITLE
PCB

### ISSUE THREAD

**Issue by danielkucera** [magnusjjj/basefx3kicad#1]
Hi,

did you also route a PCB for this schematic?

**Comment by magnusjjj** [comment]
Hi! I never wound up using it at all, sorry. I might do one for an FTDI chip in a couple of years, I guess.

I was going to make a board to listen to all the traffic between a super nintendo cartridge and the console, for figuring out some bugs in emulation. But depression hit.

**Comment by danielkucera** [comment]
Does FTDI have an alternative? USB3 to FIFO?

**Comment by magnusjjj** [comment]
Yes! the... i think.. ftdi 600 series? https://www.ftdichip.com/Products/ICs/FT600.html

On the one hand, I don't really want to give them money because of the whole ftdigate thing. On the other hand, the chip seems to be both simpler to use and way cheaper. Sadly, they don't have development boards that don't fleece you. :')

**Comment by magnusjjj** [comment]
What's your intended project? I am really new to making pcb's, in fact, my first PCB's were for this project, so, uh.. yeah. I would love to hear from you. Is it a hobby thing or otherwise? You going to open source it?

**Comment by danielkucera** [comment]
I'm working on this: https://www.thethingsnetwork.org/forum/t/lora-decoder-module/32853

but for now I think FT2232H will be sufficient.

**Comment by magnusjjj** [comment]
Yeah, that'll probably be enough bandwidth, or very nearly. What's the context? Because there are a ton of modules for arduino all over the place, and a ton of arduino compatible microcontrollers have high speed usb. I think the DUE can actually saturate the link, and I think it exists in smaller form factors.

(not raining on your parade, it's fun to build shit, just figured it's a talking point, or somewhere to start)

**Comment by danielkucera** [comment]
Hi again, I've just received my boards from JLCPCB. If you are interested, I can send you one:
https://github.com/danielkucera/cyusb3014-breakout
